src/contribution-graph/index.js

```diff
@@ -309,11 +400,12 @@ class ContributionGraph extends AbstractChart {
         (weekIndex + 1) * DAYS_IN_WEEK
       );
       const [x, y] = this.getMonthLabelCoordinates(weekIndex);
+
       return endOfWeek.getDate() >= 1 && endOfWeek.getDate() <= DAYS_IN_WEEK ? (
         <Text
           key={weekIndex}
-          x={x + paddingLeft}
```

Comment:
This `paddingLeft` is really annoying, thank god it will be removed finally.

Reply:
> This `paddingLeft` is really annoying, thank god it will be removed finally.

`paddingLeft` still exist but migrate to `chartConfig: { chartStyle:{ //here } }` without conflict with outer container `style` props.
This PR isn't going to anywhere yet, but I can guarantee the contribution chart itself it's fully functional. If you only use it you are free to try my fork. I believe Doc is up to date also.

Reply:
Great, configurable is awesome.

Reply:
I checkout to your PR, but there is no `build` script in package.json, so how can I try use it?

Reply:
> I checkout to your PR, but there is no `build` script in package.json, so how can I try use it?

The complexity comes from all the recent typescript updated. So there must be a lot of conflicts as far as I know, Since PR is all based on my fork repo, a tricky and easy way to do this is to directly `npm install` my fork repo as another library, you probably need to create alias for it so it wouldn't replace the main one, and then use the `contributionchart`, by this way you can always switch back to the main repo and delete the old one once it's ready.

Reply:
You means like?
```
git clone <your repo>
cd <your repo>
npm i
```
`npm install` in your repo won't help I think, which not install anything to my app repo.

If you're planning to create a branch new lib, a branch new pkg would be great,  otherwise, I can wait for this PR got merged (hopefully).

Reply:
No, just directly install from github repo.
check this one out for specific details: https://stackoverflow.com/questions/26414587/how-to-install-multiple-versions-of-package-using-npm/56495651#56495651
You can do something like `npm install YourOwnName@https://github.com/aboveyunhai/react-native-chart-kit.git`

then in your project
```
import { ContributionChart } from 'react-native-chart-kit';
import { ContributionChart as TempContributionChart }  from "YouOwnName";
```
Now you can use whatever you want from both repo.